### DISCUSSION TITLE
Anyone having problems with illegal offset in core extension?

### DISCUSSION THREAD

**Discussion by nzyeehaa** [datenstrom/community#649] [Ask a question] [answered]
any body have experience with this one? 

yellow 0.8.15
php 7.2

have a few websites on the same server - same setup - but one throws this error on a random basis

PHP Warning:  Illegal offset type in /home/three/public_html/.../system/extensions/core.php on line 3520

```
// Check if array element exists
    public function offsetExists($key) {
        if (is_string($key)) $key = lcfirst($key);
        return parent::offsetExists($key);  //(3520)
    }
```

**Comment by markseuffert** [comment, marked as answer]
Yes, I remember that some code was accessing `YellowArray` with a null key. The line number shows you where the problem is detected, but unfortunately not where it is caused. A stack trace should show you the calling code. For more information see https://github.com/datenstrom/yellow/discussions/567#discussioncomment-243165.

I think the problem no longer exists in the latest version of Datenstrom Yellow.

**Reply by nzyeehaa** [reply]
that explains why it happens randomly.
thanks
